internal/cmd/run.go

```diff
@@ -123,10 +148,16 @@ func runCmd() *cobra.Command {
 						blockNames[i] = block.Name()
 						blockNames[i] = blockColor.Sprint(blockNames[i])
 					}
+					if runAll && parallel {
+						blockNames = []string{blockColor.Sprint("all tasks")}
+					}
 
 					scriptRunText := "Running task"
+					if runAll && parallel {
+						scriptRunText = "Running"
+					}
```

Comment:
```suggestion
					scriptRunText := "Running task"
					if runAll && parallel {
						scriptRunText = "Running"
						blockNames = []string{blockColor.Sprint("all tasks")}
					}
```

You can bring these into one

Reply:
Good catch